utils/routes/root/routes.tsx

```diff
@@ -0,0 +1,60 @@
+// Route metadata for the profile tab
+import { Name } from './names';
+import { ParamList as TabParamList } from '../tabs/paramList';
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import { routes as tabRoutes } from '../tabs/routes';
+import { View } from 'react-native';
+
+export type Route = {
+  name: Name;
+  component: any;
+};
+
+// Style for tab bar
+const tabBarStyle = {
+  backgroundColor: 'white',
+};
+
+const BuildTabs = () => {
```

Comment:
This name should probably no longer include the "build" word, since it is now a component rather than a function that returns a component. Howabout `TabNavigator` or something similar?